lesson1/script.js

```diff
@@ -0,0 +1,2 @@
+var list = document.getElementById("js-lists");
+list.innerHTML='<li>これです</li>';
```

Comment:
ok. but 
innerHTML should not be used from a security and speed standpoint.
use `addChild` or other

Reply:
Thank you for sharing the link.  
I understand why innerHTML should not be used.